datalad/tests/test_customremotes_archive.py

```diff
@@ -72,50 +74,65 @@ def git(cmd, *args, **kwargs):
         cmd = shlex.split(cmd) if isinstance(cmd, basestring) else cmd
         return rok(["git"] + cmd, *args, **kwargs)
 
-    git('init')
-    annex('init')
-    annex('initremote annexed-archives encryption=none type=external externaltype=dl+archive')
+    handle = Handle(d, runner=r)
+    #annex('initremote annexed-archives encryption=none type=external externaltype=dl+archive')
+    handle.annex_initremote('annexed-archives',
+                                ['encryption=none', 'type=external', 'externaltype=dl+archive'])
```

Comment:
minor -- in this cases I believe it is more conventional to indent to the beginning of arguments in the prev line since no indentation is part of the syntax here.  That is what Ctrl-Enter does in PyCharm as well